### ISSUE TITLE
2024-01-05T22:55:35.894830351+01:00 [ERROR] dagger::client - timeout syncing with peer Cb1CwnBppUWfNZmeZfJoVdVzsbrJXkTHN7Fk64Q82dwy

### ISSUE THREAD

**Issue by cbijon** [GenesysGo/dagger-testnet-issues#10]
after a restart  and some minutes i have this in log :+1: 

2024-01-05T22:56:23.041035285+01:00 [ERROR] dagger::client - timeout syncing with peer 6uFd3N1H3bd7XA2bsRRFXYqTqMKY6MPzxdpocS36eDa6
2024-01-05T22:56:29.527857487+01:00 [ERROR] dagger::client - timeout syncing with peer EzxFgswVDu9S9sVZ1hM137AuqRvXudUdWzsa6t8tBEBj
2024-01-05T22:56:33.041289110+01:00 [ERROR] dagger_p2p - failed connect MaxHandshakeDurationExceeded { max_handshake_duration: 10s, source: Location { file: "/.cargo/registry/src/index.crates.io-6f17d22bba15001f/s2n-quic-transport-0.31.0/src/connection/connection_impl.rs", line: 1053, col: 24 } }
2024-01-05T22:56:39.528887579+01:00 [ERROR] dagger_p2p - failed connect MaxHandshakeDurationExceeded { max_handshake_duration: 10s, source: Location { file: "/.cargo/registry/src/index.crates.io-6f17d22bba15001f/s2n-quic-transport-0.31.0/src/connection/connection_impl.rs", line: 1053, col: 24 } }
2024-01-05T22:56:43.041914712+01:00 [ERROR] dagger_p2p - failed connect MaxHandshakeDurationExceeded { max_handshake_duration: 10s, source: Location { file: "/.cargo/registry/src/index.crates.io-6f17d22bba15001f/s2n-quic-transport-0.31.0/src/connection/connection_impl.rs", line: 1053, col: 24 } }
2024-01-05T22:56:49.529521172+01:00 [ERROR] dagger_p2p - failed connect MaxHandshakeDurationExceeded { max_handshake_duration: 10s, source: Location { file: "/.cargo/registry/src/index.crates.io-6f17d22bba15001f/s2n-quic-transport-0.31.0/src/connection/connection_impl.rs", line: 1053, col: 24 } }
2024-01-05T22:56:51.192948206+01:00 [INFO] dagger::p2p::graph::dagger - Added node to DAGGER in 0 millis; new size is 600 w/ current nodes 600
2024-01-05T22:56:53.042249946+01:00 [ERROR] dagger_p2p - failed connect MaxHandshakeDurationExceeded { max_handshake_duration: 10s, source: Location { file: "/.cargo/registry/src/index.crates.io-6f17d22bba15001f/s2n-quic-transport-0.31.0/src/connection/connection_impl.rs", line: 1053, col: 24 } }
2024-01-05T22:57:02.005414263+01:00 [ERROR] dagger_p2p - failed connect MaxHandshakeDurationExceeded { max_handshake_duration: 10s, source: Location { file: "/.cargo/registry/src/index.crates.io-6f17d22bba15001f/s2n-quic-transport-0.31.0/src/connection/connection_impl.rs", line: 1053, col: 24 } }
2024-01-05T22:57:04.359826897+01:00 [ERROR] dagger_p2p - failed connect MaxHandshakeDurationExceeded { max_handshake_duration: 10s, source: Location { file: "/.cargo/registry/src/index.crates.io-6f17d22bba15001f/s2n-quic-transport-0.31.0/src/connection/connection_impl.rs", line: 1053, col: 24 } }
2024-01-05T22:57:14.708220680+01:00 [ERROR] dagger_p2p - failed connect MaxHandshakeDurationExceeded { max_handshake_duration: 10s, source: Location { file: "/.cargo/registry/src/index.crates.io-6f17d22bba15001f/s2n-quic-transport-0.31.0/src/connection/connection_impl.rs", line: 1053, col: 24 } }
2024-01-05T22:57:15.639630245+01:00 [ERROR] dagger_p2p - failed connect MaxHandshakeDurationExceeded { max_handshake_duration: 10s, source: Location { file: "/.cargo/registry/src/index.crates.io-6f17d22bba15001f/s2n-quic-transport-0.31.0/src/connection/connection_impl.rs", line: 1053, col: 24 } }
2024-01-05T22:57:17.099907780+01:00 [INFO] dagger::p2p::graph::dagger - Added node to DAGGER in 0 millis; new size is 700 w/ current nodes 700
2024-01-05T22:57:25.640665895+01:00 [ERROR] dagger_p2p - failed connect MaxHandshakeDurationExceeded { max_handshake_duration: 10s, source: Location { file: "/.cargo/registry/src/index.crates.io-6f17d22bba15001f/s2n-quic-transport-0.31.0/src/connection/connection_impl.rs", line: 1053, col: 24 } }

**Comment by kintincheung** [comment]
Having the same, but with 3 more peers:

`2024-01-06T17:12:37.164003077+01:00 [ERROR] dagger::client - timeout syncing with peer 9WDbcQG5yvg2VAFXiSAnJoAbnD4D3SMx1YQUrLiUHvKy
2024-01-06T17:12:46.072163104+01:00 [ERROR] dagger::client - timeout syncing with peer EzxFgswVDu9S9sVZ1hM137AuqRvXudUdWzsa6t8tBEBj
2024-01-06T17:12:47.165181755+01:00 [ERROR] dagger::client - timeout syncing with peer 6uFd3N1H3bd7XA2bsRRFXYqTqMKY6MPzxdpocS36eDa6
2024-01-06T17:12:56.073604999+01:00 [ERROR] dagger::client - timeout syncing with peer Cb1CwnBppUWfNZmeZfJoVdVzsbrJXkTHN7Fk64Q82dwy
2024-01-06T17:12:57.166732705+01:00 [ERROR] dagger::client - timeout syncing with peer EzxFgswVDu9S9sVZ1hM137AuqRvXudUdWzsa6t8tBEBj
2024-01-06T17:33:01.110751556+01:00 [WARN] dagger::client - Long sync with peer 9WDbcQG5yvg2VAFXiSAnJoAbnD4D3SMx1YQUrLiUHvKy: 5404 millis
2024-01-06T19:37:34.483242900+01:00 [WARN] dagger::client - Long sync with peer ARfg7k3wTR43W9pqNhNVsSAvazDfvXyNNNVyKjn4GwaN: 6526 millis
2024-01-06T19:38:01.726899218+01:00 [WARN] dagger::client - Long sync with peer ARfg7k3wTR43W9pqNhNVsSAvazDfvXyNNNVyKjn4GwaN: 6164 millis
2024-01-06T19:38:17.149409802+01:00 [WARN] dagger::client - Long sync with peer EzxFgswVDu9S9sVZ1hM137AuqRvXudUdWzsa6t8tBEBj: 8036 millis
2024-01-06T20:46:45.448591189+01:00 [WARN] dagger::client - Long sync with peer 6uFd3N1H3bd7XA2bsRRFXYqTqMKY6MPzxdpocS36eDa6: 5424 millis`

**Comment by kintincheung** [comment]
Fixed it by reinstalling with the startup script

**Comment by kintincheung** [comment]
> Fixed it by reinstalling with the startup script

Problem keeps coming back

**Comment by OlajideOgun** [comment]
Having same issue

```Jan 11 01:14:51 taiwebapp start_wield.sh[420490]: joining in on epoch 8754
Jan 11 01:14:51 taiwebapp start_wield.sh[420490]: 2024-01-11T01:14:51.688501470+00:00 [INFO] dagger::p2p::join - Attempting to join network with 'trusted' peer: 184.1>
Jan 11 01:14:51 taiwebapp start_wield.sh[420490]: 2024-01-11T01:14:51.823239644+00:00 [INFO] dagger::p2p::join - Connected to trusted peer
Jan 11 01:14:51 taiwebapp start_wield.sh[420490]: 2024-01-11T01:14:51.823314275+00:00 [INFO] dagger::p2p::join - Requested peer list from trusted peer
Jan 11 01:14:52 taiwebapp start_wield.sh[420490]: 2024-01-11T01:14:52.090900196+00:00 [INFO] dagger::p2p::join - Obtained peer list from trusted peer
Jan 11 01:14:52 taiwebapp start_wield.sh[420490]: 2024-01-11T01:14:52.097909694+00:00 [INFO] dagger::p2p::join - Successfully joined network
Jan 11 01:14:52 taiwebapp start_wield.sh[420490]: 2024-01-11T01:14:52.098148901+00:00 [INFO] dagger::p2p::graph::dagger - entering dagger loop
Jan 11 01:14:52 taiwebapp start_wield.sh[420490]: 2024-01-11T01:14:52.098608436+00:00 [INFO] dagger::controller - entering controller loop
Jan 11 01:14:52 taiwebapp start_wield.sh[420490]: 2024-01-11T01:14:52.101842302+00:00 [INFO] dagger::client - joined with peer list {Peer { pubkey: 6Fv9bYXVK6HgspzJye>
Jan 11 01:14:52 taiwebapp start_wield.sh[420490]: 2024-01-11T01:14:52.105036844+00:00 [INFO] dagger::client - entering conn loop
Jan 11 01:14:52 taiwebapp start_wield.sh[420490]: 2024-01-11T01:14:52.105215804+00:00 [INFO] dagger::client - entering sync loop
Jan 11 01:14:52 taiwebapp start_wield.sh[420490]: 2024-01-11T01:14:52.106213622+00:00 [INFO] dagger::processor - entering processor loop
Jan 11 04:08:04 taiwebapp start_wield.sh[420490]: 2024-01-11T04:08:04.049270756+00:00 [ERROR] dagger::client - timeout syncing with peer EzxFgswVDu9S9sVZ1hM137AuqRvXu>
Jan 11 04:08:15 taiwebapp start_wield.sh[420490]: 2024-01-11T04:08:15.199734365+00:00 [ERROR] dagger::client - timeout syncing with peer EzxFgswVDu9S9sVZ1hM137AuqRvXu>
Jan 11 04:08:28 taiwebapp start_wield.sh[420490]: 2024-01-11T04:08:28.889292772+00:00 [WARN] dagger::client - Long sync with peer EzxFgswVDu9S9sVZ1hM137AuqRvXudUdWzsa>
Jan 11 04:08:41 taiwebapp start_wield.sh[420490]: 2024-01-11T04:08:41.974356138+00:00 [ERROR] dagger::client - timeout syncing with peer EzxFgswVDu9S9sVZ1hM137AuqRvXu>
Jan 12 10:30:56 taiwebapp start_wield.sh[420490]: 2024-01-12T10:30:56.897886415+00:00 [WARN] dagger::client - Long sync with peer 9WDbcQG5yvg2VAFXiSAnJoAbnD4D3SMx1YQU>
Jan 13 21:01:27 taiwebapp start_wield.sh[420490]: 2024-01-13T21:01:27.209276037+00:00 [WARN] dagger::client - Long sync with peer Cb1CwnBppUWfNZmeZfJoVdVzsbrJXkTHN7Fk>
Jan 14 12:54:02 taiwebapp start_wield.sh[420490]: 2024-01-14T12:54:02.428124356+00:00 [WARN] dagger::client - Long sync with peer 9WDbcQG5yvg2VAFXiSAnJoAbnD4D3SMx1YQU>```

**Comment by fetith** [comment]
similar issue.
I hope my logs can help the team fix it.
[config.log](https://github.com/GenesysGo/dagger-testnet-issues/files/13936067/config.log)

As additional information, all peer IPs when the following error occurred were 184.154.98.***.
  [ERROR] dagger::client - timeout syncing with peer
When searching on Discord, I found someone who was also receiving errors only from the same IP range.

**Comment by fetith** [comment]
Only Peers for which hw_info cannot be obtained seem to return "[ERROR] dagger::client - timeout syncing with peer".
<img width="620" alt="image" src="https://github.com/GenesysGo/dagger-testnet-issues/assets/67474560/a8a86380-25c4-4a9e-923f-220e5d204c12">

**Comment by fetith** [comment]
The problem in my environment seems to be solved in 0.7.4.
I appreciate the team.

**Comment by OlajideOgun** [comment]
my problem is fixed in 0.7.4.

**Comment by tracy-codes** [comment]
Closed as a duplicate of #4
